enclave-runtime/src/top_pool_execution.rs

```diff
@@ -271,52 +261,18 @@ where
 
 	let mut aura = Aura::<_, ParentchainBlock, SignedSidechainBlock, PEnvironment, _, _>::new(
 		authority,
-		ocall_api,
+		ocall_api.as_ref().clone(),
 		block_import_trigger,
 		proposer_environment,
 	)
 	.with_claim_strategy(SlotClaimStrategy::RoundRobin)
 	.with_allow_delayed_proposal(false);
 
-	let (blocks, xts): (Vec<_>, Vec<_>) =
+	let (blocks, _xts): (Vec<_>, Vec<_>) =
 		PerShardSlotWorkerScheduler::on_slot(&mut aura, slot, shards)
 			.into_iter()
 			.map(|r| (r.block, r.parentchain_effects))
 			.unzip();
 
-	let opaque_calls: Vec<OpaqueCall> = xts.into_iter().flatten().collect();
-	Ok((blocks, opaque_calls))
-}
-
-/// Gossips sidechain blocks to fellow peers and sends opaque calls as extrinsic to the parentchain.
-pub(crate) fn send_blocks_and_extrinsics<
```

Comment:
This won't work like this, because there are parentchain extrinsic different from sidechain block confirmations, and SDK users depend on these extra extrinsics.

Reply:
I think we can put this back in, for now. So that the extrinsics are still being send.

I also think we should discuss how communication between sidechain and parentchain works ... this needs a design task first, I would say.

@haerdib and @mullefel : if you agree I put the code back in. Or do you want to do this @mullefel ? (Since it's your code)

Reply:
Yes, from my side: please create an issue (with the mention of the test) for this and readd the extrinsic sending

Reply:
As far as I could tell, we don't actually make use of these extra extrinsics. Only the confirmation was ever included in this extrinsic. So IMO we don't break anything if we remove it now.

Reply:
Let's not break something unnecessarily, be it in use or not. So readd. Issue is necessary nonetheless.